modules/alloy/src/main/ts/ephox/alloy/aria/AriaFocus.ts

```diff
@@ -13,7 +13,7 @@ const preserve = <T>(f: (e: SugarElement) => T, container: SugarElement): T => {
 
   // If there is a focussed element, the F function may cause focus to be lost (such as by hiding elements). Restore it afterwards.
   refocus.each((oldFocus: SugarElement) => {
-    Focus.active(ownerDoc).filter((newFocus) => Compare.eq(newFocus, oldFocus)).fold(() => {
+    Focus.active(dos).filter((newFocus) => Compare.eq(newFocus, oldFocus)).fold(() => {
```

Comment:
Small thing but is this meant to be "doc"?

Reply:
Reading on I'm thinking not... What are we meaning by "dos". Shadow DOM document?

Reply:
Convention started by @tinydylan... it means `DocumentOrShadowRoot` I believe.

Reply:
Close. `Element<Document | ShadowRoot>`. Might be less confusing if we call it `root`.